websites/E/Emby/presence.ts

```diff
@@ -355,23 +366,43 @@ async function isEmbyWebClient(): Promise<boolean> {
 	return false;
 }
 
+function embyBasenameUrl(): string {
+	return `${`${location.protocol}//${location.host}${location.pathname.replace(
+		location.pathname.split("/").slice(-2).join("/"),
+		""
+	)}`}`;
+}
+
+function mediaPrimaryImage(mediaId: string): string {
```

Comment:
It's a bit tricky, the one who uses the presence will always be able to see the image.
Users that open the profile will see a broken image, same happens if its an internal IP (10.0.0.0, 172.16.0.0, 192.168.0.0) or any non globally resolvable domain.

Reply:
Can't you upload the image to the premid CDN? https://discord.com/channels/493130730549805057/607524579874832446/928096461994987540

Reply:
It would change nothing, the URL has to be reachable for the CDN. It does not support sending a file. So it does not make any sense, also the URLs do not get over the 256 chars limit